aqua-examples/near-integration/README.md

```diff
@@ -270,7 +270,7 @@ In order to create a NEAR Wasm adapter, we wrap whatever functionality we need f
 
 ### Creating And Deploying NEAR Wasm Services
 
-In the `services` directory, you find a minimal Wasm adapter for [NEAR RPC API](https://docs.near.org/docs/api/rpc) to get you started. Since we are connecting to on-chain resources via JSON-RPC, we need our service module to have access to [cUrl](https://doc.fluence.dev/docs/tutorials_tutorials/curl-as-a-service), which we provide with the [cUrl adapter](../near-integration/services/curl-adapter/):
+In the `services` directory, you find a minimal Wasm adapter for [NEAR RPC API](https://docs.near.org/docs/api/rpc) to get you started. Since we are connecting to on-chain resources via JSON-RPC, we need our service module to have access to [cUrl](https://fluence.dev/docs/build/tutorials/curl-as-a-service), which we provide with the [cUrl adapter](../near-integration/services/curl-adapter/):
```

Comment:
looks like `https://docs.near.org/docs/api/rpc` now redirects to `https://docs.near.org/api/rpc/introduction`.

Reply:
fixed